packages/fuels-abigen-macro/tests/harness.rs

```diff
@@ -1078,8 +1015,10 @@ async fn test_large_return_data() {
         ]
     );
 
-    let res = contract_instance.get_small_string().call().await.unwrap();
-    assert_eq!(res.value, "gggggggg");
+    // TODO https://github.com/FuelLabs/fuels-rs/issues/201
+    // One word-sized string
+    // let res = contract_instance.get_small_string().call().await.unwrap();
+    // assert_eq!(res.value, "gggggggg");
```

Comment:
```suggestion
    // One word-sized string
    let res = contract_instance.get_small_string().call().await.unwrap();
    assert_eq!(res.value, "gggggggg");
```

This should be working now.